scene/main/node.cpp

```diff
@@ -2620,7 +2619,6 @@ Node *Node::_duplicate(int p_flags, HashMap<const Node *, Node *> *r_duplimap) c
 			parent->move_child(dup, pos);
 		}
 	}
-
 	for (List<const Node *>::Element *N = node_tree.front(); N; N = N->next()) {
```

Comment:
```suggestion

	for (List<const Node *>::Element *N = node_tree.front(); N; N = N->next()) {
```
Unnecessary and unrelated whitespace change